css/style.css

```diff
@@ -0,0 +1,700 @@
+  color: var(--clr-dark);
+  font-size: 14px;
+  padding: 10px 20px 40px;
+  text-align: center;
+}
+
+.email_link {
+  text-decoration: underline;
+  color: var(--clr-dark);
+  font-weight: bolder;
+}
+
+.section_two {
+  padding: 30px 20px;
+  font-family: var(--ff-lato);
+  color: var(--clr-dark);
+  text-align: center;
+}
+
+.logo_line {
+  width: 30px;
+}
+
+.section_two p,
+.section_three p {
+  padding-top: 30px;
+  font-size: 16px;
+}
+
+.section_two img {
+  margin-top: 30px;
+  max-width: 240px;
+  height: auto;
+  padding: 15px 30px;
+  border: 1px solid var(--clr-gray);
+}
+
+.divider {
+  display: block;
+  width: 100%;
+  height: 1px;
+  background-color: var(--clr-gray);
+  margin: 20px 0;
+}
+
+.section_three {
+  padding: 30px 20px;
+  font-family: var(--ff-lato);
+  color: var(--clr-dark);
+  text-align: center;
+}
+
+.section_three .part1 {
+  padding: 40px 20px;
+  color: white;
+  margin-top: 30px;
+  background-image: url("../assets/concert.jpg");
+  background-size: cover;
+  box-shadow: inset 0 0 0 2000px rgba(200, 67, 53, 0.55);
+  transition: 0.8s;
+}
+
+.section_three .part2 {
+  padding: 40px 20px;
+  color: white;
+  margin-top: 30px;
+  background-image: url("../assets/concert2.jpg");
+  background-size: cover;
+  background-position-y: -50px;
+  box-shadow: inset 0 0 0 2000px rgba(200, 67, 53, 0.55);
+  transition: 0.8s;
+}
+
+.concert:hover {
+  cursor: pointer;
+  box-shadow: inset 0 0 0 2000px rgba(200, 67, 53, 0.85);
+  color: var(--clr-gray);
+}
+
+.concert h3 {
+  font-size: 17px;
+}
+
+/* Desktop Version */
+
+@media (min-width: 768px) {
+  nav .fa-bars {
+    display: none;
+  }
+
+  .navbar ul a {
+    text-decoration: none;
+    font-family: var(--ff-lato);
+    color: var(--clr-dark);
+    font-size: 16px;
+  }
+
+  .top li a {
+    text-decoration: none;
+    color: white;
+    font-family: var(--ff-lato);
+    font-size: 15px;
+  }
+
+  .navbar .last a {
+    border: 2px solid var(--clr-red);
+    color: var(--clr-red);
+    padding: 5px 10px;
+    margin-top: -10px;
+  }
+
+  .navbar .active a {
+    color: var(--clr-red);
+    border-bottom: 1px solid var(--clr-red);
+    padding-bottom: 3px;
+  }
+
+  .navbar ul a:hover {
+    transition: 0.2s ease-in;
+    color: var(--clr-red);
+  }
+
+  .top {
+    padding: 5px 150px;
+    display: flex;
+    justify-content: flex-end;
+    gap: 20px;
+    background-color: var(--clr-dark);
+    color: white;
+  }
+
+  .top li {
+    list-style-type: none;
+    display: inline-block;
+    padding: 0 5px;
+  }
+
+  .top li a:hover,
+  .top i:hover {
+    color: var(--clr-red);
+    cursor: pointer;
+    transition: 0.2s ease;
+  }
+
+  .navbar {
+    display: flex;
+    background-color: white;
+    justify-content: space-between;
+    padding: 15px 10vw;
+  }
+
+  .navbar img {
+    max-width: 150px;
+    height: auto;
+  }
+
+  .navbar ul {
+    padding-top: 5px;
+    display: flex;
+    list-style-type: none;
+    justify-self: flex-end;
+    gap: 50px;
+  }
```

Comment:
- [ ] On screen sizes close to the transition from desktop to mobile(768px) the 'Get Connected' button breaks. I recommend you reduce the `gap` so it can look consistent in all screen sizes.
 - Your design:
 - 
<img width="867" alt="image" src="https://user-images.githubusercontent.com/61919617/203641107-1b3db8ff-0a2a-4eef-9041-9d9017195015.png">